content/synchronization_model.tex

```diff
@@ -19,9 +19,10 @@
 \begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
 {}
 &
-{ Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
-used when computation on the local \ac{PE} cannot proceed without the value that
-the remote \ac{PE} is to update.} \tabularnewline
+{ Waits for a symmetric variable to be updated by a remote \ac{PE} \newtext{or the same
+\ac{PE} in the case of multi-threaded \openshmem{}}. Should be
+used when computation on the \newtext{calling thread} \oldtext{local \ac{PE}} cannot proceed without the value that
+the remote \newtext{thread or} \ac{PE} is to update.} \tabularnewline
```

Comment:
@nspark @shamisp I will remove new changes to this section (8.9.3). It doesn't require update in thread ticket. However, it might be more useful to cleanup as a whole, and add new examples. What do you think ?